backend/routers/medical_documentation.py

```diff
@@ -0,0 +1,58 @@
+from fastapi import APIRouter, Depends, Header, HTTPException, Query
+from typing import Annotated
+from models.database import SessionLocal
+import models.medical_documentation as models
+import schemas.medical_documentation as schemas
+from sqlalchemy.orm import Session, joinedload
+
+router = APIRouter()
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+async def create_medical_documentation_if_not_exists(db: Session, user_id: int) -> models.MedicalDocumentation:
+    medical_documentation = db.query(models.MedicalDocumentation).filter(models.MedicalDocumentation.patient_id == user_id).first()
+    if medical_documentation is None:
+        medical_documentation = models.MedicalDocumentation(patient_id=user_id)
+        db.add(medical_documentation)
+        db.commit()
+        
+    return medical_documentation
+    
+@router.get("/medical_documentation")
+async def get_medical_documentation(user_id: int = Query(None), db: Session = Depends(get_db)):
+    if user_id is None:
+        raise HTTPException(status_code=400, detail="ID not provided")
+    medical_documentation = db.query(models.MedicalDocumentation).filter(models.MedicalDocumentation.patient_id == user_id).first()
+    if medical_documentation is None:
+        raise HTTPException(status_code=404, detail="User not found")
+    return medical_documentation
+    
+
+@router.get("/medical_documentation/{medical_documentation_entry_id}")
```

Comment:
I think we should standardize medical_documentations endpoints urls.
1.  Use plural form, change '_' to '-'. [Source](https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5)
2. Medical documentations have their unique ID. Make MedicalDocumentations primary key also a foreign key pointing to user or use the unique ID. Either way, the {parameter} in this road should be MedicalDocumentation ID, not a MedicalDocumentationEntry ID. Same applies to every other medical documentation endpoint.
3. We cannot neglect doctors' responsibilities. User should only see his own documentation, doctors should be able to see all of them. That is why we should make endpoints at least for: listing documentation `medical-documentations/`, getting one documentation entries medical-documentations/{medical-documentations-id}/entries. 

-